src/Components/Common/Button/Button.js

```diff
@@ -1,7 +1,8 @@
 import classes from './Button.module.scss';
+import cn from 'classnames';
 
-export const Button = ({text, onClick, margin}) => {
+export const Button = ({text, onClick, extraClass}) => {
   return (
-    <button onClick={onClick} className={classes.button} style={margin}>{text}</button>
+    <button onClick={onClick} className={cn(classes.button, extraClass)}>{text}</button>
```

Comment:
No need to call it like "extra", let's use just "className" =)
There are no any extras here, just regular class name =)

Reply:
Done!